hibike/unit_tests/hibike_message_tests.py

```diff
@@ -0,0 +1,386 @@
+"""
+Unit tests for functions in hibike_message.
+"""
+
+
+import unittest
+import random
+import queue
+import time
+import threading
+
+import serial
+
+import hibike_message
+import spawn_virtual_devices
+DEVICE_TYPES = list(hibike_message.DEVICES)
```

Comment:
Why convert this to a list? `hibike_message.devices` is originally a dictionary that should have all the info you need. The way youre using `DEVICE_TYPES` later also suggests you want access to that data